apacheconfig/host.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+class LocalHost(object):
```

Comment:
Would smth like `LocalHostReader` name be a tad bit more descriptive?

Reply:
Yeah, I do think the name LocalHost is a bit ambiguous, I just wasn't sure about reader either because it's doing a bit more than just reading a file. I will think about what to name it, but I'm also happy with naming it *Reader if you think that's a good name.